workflows/microbiome/gene-based-pathogen-identification/Gene-based-Pathogen-Identification.ga

```diff
@@ -0,0 +1,1165 @@
+    "annotation": "Nanopore datasets analysis - Phylogenetic Identification - ST - antibiotic resistance genes detection and contigs building",
+    "comments": [
+        {
+            "child_steps": [
+                11,
+                14,
+                15,
+                17
+            ],
+            "color": "turquoise",
+            "data": {
+                "title": "Identifying AMR"
+            },
+            "id": 3,
+            "position": [
+                1830.3,
+                602.8000000000001
+            ],
+            "size": [
+                1198.5,
+                285.6
+            ],
+            "type": "frame"
+        },
+        {
+            "child_steps": [
+                10,
+                13
+            ],
+            "color": "turquoise",
+            "data": {
+                "title": "Identifying VF"
+            },
+            "id": 2,
+            "position": [
+                1821.7,
+                0.6000000000000014
+            ],
+            "size": [
+                461.7,
+                234.8
+            ],
+            "type": "frame"
+        },
+        {
+            "child_steps": [
+                2,
+                4,
+                6,
+                7,
+                9,
+                12,
+                16
+            ],
+            "color": "turquoise",
+            "data": {
+                "title": "Extracting Contigs from our Samples"
+            },
+            "id": 1,
+            "position": [
+                310,
+                490.20000000000005
+            ],
+            "size": [
+                1468.9,
+                676
+            ],
+            "type": "frame"
+        },
+        {
+            "child_steps": [
+                1,
+                3,
+                5,
+                8
+            ],
+            "color": "turquoise",
+            "data": {
+                "title": "Extract Samples IDs"
+            },
+            "id": 0,
+            "position": [
+                302.4,
+                0.0
+            ],
+            "size": [
+                1060.5,
+                234.8
+            ],
+            "type": "frame"
+        }
+    ],
+    "creator": [
+        {
+            "class": "Person",
+            "identifier": "0000-0001-9047-4215",
+            "name": "Engy Nasr",
+            "url": "https://orcid.org/0000-0001-9047-4215"
+        },
+        {
+            "class": "Person",
+            "identifier": "0000-0001-9852-1987",
+            "name": "B\u00e9r\u00e9nice Batut",
+            "url": "https://orcid.org/0000-0001-9852-1987"
+        },
+        {
+            "class": "Person",
+            "identifier": "0000-0003-2982-388X",
+            "name": "Paul Zierep",
+            "url": "https://orcid.org/0000-0003-2982-388X"
+        }
+    ],
+    "format-version": "0.1",
+    "license": "MIT",
+    "release": "0.1",
+    "name": "Gene-based-Pathogen-Identification",
```

Comment:
Please use the same name as in the README file. Same for other workflows

Reply:
its Gene based pathogen identification everywhere, I will also recheck

Reply:
I meant without the hyphens `-`